mysql-async/src/main/java/com/github/jasync/sql/db/mysql/MySQLConnection.kt

```diff
@@ -242,7 +242,7 @@ class MySQLConnection @JvmOverloads constructor(
   }
 
   override fun disconnect(): CompletableFuture<Connection> = this.close()
-  override fun onTimeout(): CompletableFuture<Connection> = disconnect()
+  override fun onTimeout() {disconnect()}
```

Comment:
its better to leave return value so should be 
`override fun onTimeout(): Unit = disconnect()`